lib/vyasa_web/components/layouts/app.html.heex

```diff
@@ -23,6 +23,6 @@
     <.flash_group flash={@flash} />
   </div>
   <!-- DM manages the core layouts -->
-  <%= live_render(@socket, VyasaWeb.DisplayManager.DisplayManager, id: "DisplayManager", session: @session, sticky: true) %>
+  <%= live_render(@socket, VyasaWeb.DisplayManager.DisplayLive, id: "DisplayManager", session: @session, sticky: true) %>
```

Comment:
so DM is a live_view that mediates everything else. 

nesting liveviews within it seems to be harmless, didn't encounter mounting issues or race conditions so far

Reply:
nesting livecomponents* but liveview to livecomponents is clean 

nested liveviews to a depth more than 2 > was causing issues with nested stickies for us